static/app/views/settings/project/projectOwnership/ruleBuilder.tsx

```diff
@@ -169,29 +173,26 @@ class RuleBuilder extends Component<Props, State> {
               placeholder="tag-name"
             />
           )}
-          <BuilderInput
+          <Input
             value={text}
             onChange={this.handleChangeValue}
             disabled={disabled}
             placeholder={getMatchPlaceholder(type)}
+            aria-label={t('Rule pattern')}
           />
-          <Divider direction="right" />
-          <SelectOwnersWrapper>
-            <SelectOwners
-              organization={organization}
-              project={project}
-              value={owners}
-              onChange={this.handleChangeOwners}
-              disabled={disabled}
-            />
-          </SelectOwnersWrapper>
-
-          <AddButton
+          <IconChevron color="border" direction="right" />
+          <SelectOwners
+            organization={organization}
+            project={project}
+            value={owners}
+            onChange={this.handleChangeOwners}
+            disabled={disabled}
+          />
+          <Button
             priority="primary"
             disabled={!isValid}
             onClick={this.handleAddRule}
             icon={<IconAdd isCircled />}
-            size="sm"
```

Comment:
is this the default?

Reply:
the default is `md` but we were adding padding to the button to make it size md, so I just.. let it be size md